PaddleAudio/examples/dcase2021_task1b/model.py

```diff
@@ -0,0 +1,409 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Is there any difference between `paddle.vision.ResNet` ? https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/vision/Overview_cn.html

Reply:
Yes, some major differences:  1. added batchnorm to normalize along mel-bins, as in PANNS, 2. added dropout between each layer block, and  3. the classification head is different as we are doing classification for ten classes.